library-udf/src/test/java/org/apache/iotdb/library/dmatch/DMatchTests.java

```diff
@@ -0,0 +1,547 @@
+      int resultSetLength = resultSet.getRow();
+      assert resultSetLength == 1;
+      Double result = Double.parseDouble(resultSet.getString(1));
+      assert result >= 0.0D;
+    } catch (SQLException throwable) {
+      fail(throwable.getMessage());
+    }
+  }
+
+  @Test
+  public void testDtw2() {
+    String sqlStr = "select dtw(d1.s2,d1.s4) from root.vehicle";
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      ResultSet resultSet = statement.executeQuery(sqlStr);
+      resultSet.last();
+      int resultSetLength = resultSet.getRow();
+      assert resultSetLength == 1;
+      Double result = Double.parseDouble(resultSet.getString(1));
+      assert result >= 0.0D;
+    } catch (SQLException throwable) {
+      fail(throwable.getMessage());
+    }
+  }
+
+  @Test
+  public void testDtw3() {
+    String sqlStr = "select dtw(d2.s1,d2.s3) from root.vehicle";
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      ResultSet resultSet = statement.executeQuery(sqlStr);
+      resultSet.last();
+      int resultSetLength = resultSet.getRow();
+      assert resultSetLength == 1;
+      Double result = Double.parseDouble(resultSet.getString(1));
+      assert result >= 0.0D;
+    } catch (SQLException throwable) {
+      fail(throwable.getMessage());
+    }
+  }
+
+  @Test
+  public void testDtw4() {
+    String sqlStr = "select dtw(d2.s2,d2.s4) from root.vehicle";
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      ResultSet resultSet = statement.executeQuery(sqlStr);
+      resultSet.last();
+      int resultSetLength = resultSet.getRow();
+      assert resultSetLength == 1;
+      Double result = Double.parseDouble(resultSet.getString(1));
+      assert result >= 0.0D;
+    } catch (SQLException throwable) {
+      fail(throwable.getMessage());
+    }
+  }
+
+  @Test
+  public void testPearson1() {
+    String sqlStr = "select pearson(d1.s1,d1.s3) from root.vehicle";
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      ResultSet resultSet = statement.executeQuery(sqlStr);
+      Double result = Double.parseDouble(resultSet.getString(1));
+      assert result >= -0.0D && result <= 1.0D;
+    } catch (SQLException throwable) {
+      fail(throwable.getMessage());
+    }
+  }
+
+  @Test
+  public void testPearson2() {
+    String sqlStr = "select pearson(d1.s2,d1.s4) from root.vehicle";
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      ResultSet resultSet = statement.executeQuery(sqlStr);
+      Double result = Double.parseDouble(resultSet.getString(1));
+      assert result >= -0.0D && result <= 1.0D;
```

Comment:
Please use `Assert.assertEqual()` instead of `assert` keyword.
Is this result not deterministic? If the result is consistent of in every test running, please check the equalivent instead of a range condition check.

Reply:
Done.